models/vision_cam/model.sdf

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" ?>
+<sdf version="1.4">
+  <model name="camera">
+    <link name="link">
+      <inertial>
+        <pose>0.01 0.025 0.025 0 -0 0</pose>
+        <mass>0.01</mass>
+        <inertia>
+          <ixx>4.15e-6</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>2.407e-6</iyy>
+          <iyz>0</iyz>
+          <izz>2.407e-6</izz>
+        </inertia>
+      </inertial>
+      <visual name="visual">
+        <geometry>
+          <box>
+            <size>0.02 0.05 0.05</size>
+          </box>
+        </geometry>
+      </visual>
+      
+      <!--752x480 MT9V034 image sensor-->
+      <sensor name="vision_cam" type="camera">
+        <always_on>true</always_on>
+        <update_rate>40.0</update_rate>
+        <visualize>true</visualize>
+        <camera>
+          <horizontal_fov>1.0</horizontal_fov>
+          <image>
+            <width>752</width>
+            <height>480</height>
+            <format>L8</format>
+          </image>
+          <clip>
+            <near>0.1</near>
+            <far>100</far>
+          </clip>
+          <noise>
+            <type>gaussian</type>
+            <mean>0.0</mean>
+            <stddev>0.001</stddev>
+          </noise>
+        </camera>
+        <plugin name="svo_plugin" filename="libgazebo_ros_camera.so">
```

Comment:
`svo_plugin`?

Reply:
I've tested this model with external vision library which is called svo: https://github.com/uzh-rpg/rpg_svo. But it's up to user what name for plugin is preffered. We may call it vision_plugin.